airflow/models/taskinstance.py

```diff
@@ -83,7 +83,6 @@
     AirflowTaskTimeout,
     DagRunNotFound,
     RemovedInAirflow3Warning,
-    TaskDeferralError,
```

Comment:
Can we remove this? (Backward compatibility)

Reply:
This is fixed while rebasing now.